js/libs.js

```diff
@@ -75,6 +75,8 @@ Fliplet.Registry.set('notification-inbox:1.0:core', function(element, data) {
       $notifications.after($loadMore);
     }
 
+    $('.notification-read').hide();
```

Comment:
This line tells me that if when a notification is rendered, after it's rendered, always hide the notifications that are read. Shouldn't it only do that if the toggle says "Only show unread"? See the following acceptance criteria.

> If toggle is active to only show unread => Notification list is refreshed after a notifications goes from unread to read (Read notification no longer displayed)